src/card_operations/card_fetch_share.c

```diff
@@ -164,8 +164,13 @@ card_error_type_e card_fetch_share(card_fetch_share_cfg_t *config) {
       }
     }
 
-    if ((CARD_OPERATION_CARD_REMOVED == card_data.error_type) ||
-        (CARD_OPERATION_RETAP_BY_USER_REQUIRED == card_data.error_type)) {
+    if (CARD_OPERATION_CARD_REMOVED == card_data.error_type) {
+      continue;
+    } else {
+      buzzer_start(BUZZER_DURATION);
```

Comment:
Can you review if the buzzer can be moved outside the while 1 loop altogether?

In that case, we can remove it from the success case as well.. and have it in one common place